### ISSUE TITLE
missing "_license" key for gnomad and exac_nontcga

### ISSUE THREAD

**Issue by newgene** [biothings/myvariant.info#88]
Here is an example:

http://myvariant.info/v1/variant/chr6:g.38906659G%3EA?fields=gnomad*.af,exac*.af

Should add this to the tests.

**Comment by namespacestd0** [comment]
Implemented in 6ffafdfc37575c0b99c2e620abc08b717aac5e0f and a417542f168bef8e9294939572b4fac8402d5c52.

**Comment by namespacestd0** [comment]
Deployed. The referenced link returns this now:
```
{
  "_id": "chr6:g.38906659G>A",
  "_version": 2,
  "exac": {
    "_license": "http://bit.ly/2H9c4hg",
    "af": 2.471e-05
  },
  "exac_nontcga": {
    "_license": "http://bit.ly/2H9c4hg",
    "af": 1.883e-05
  },
  "gnomad_exome": {
    "_license": "http://bit.ly/2I1cl1I",
    "af": {
      "af": 1.19429e-05,
      "af_afr": 0.000123077,
      "af_afr_female": 9.93443e-05,
      "af_afr_male": 0.000161708,
      "af_amr": 0.0,
      "af_amr_female": 0.0,
      "af_amr_male": 0.0,
      "af_asj": 0.0,
      "af_asj_female": 0.0,
      "af_asj_male": 0.0,
      "af_eas": 0.0,
      "af_eas_female": 0.0,
      "af_eas_jpn": 0.0,
      "af_eas_kor": 0.0,
      "af_eas_male": 0.0,
      "af_eas_oea": 0.0,
      "af_female": 8.66296e-06,
      "af_fin": 0.0,
      "af_fin_female": 0.0,
      "af_fin_male": 0.0,
      "af_male": 1.47317e-05,
      "af_nfe": 0.0,
      "af_nfe_bgr": 0.0,
      "af_nfe_est": 0.0,
      "af_nfe_female": 0.0,
      "af_nfe_male": 0.0,
      "af_nfe_nwe": 0.0,
      "af_nfe_onf": 0.0,
      "af_nfe_seu": 0.0,
      "af_nfe_swe": 0.0,
      "af_oth": 0.0,
      "af_oth_female": 0.0,
      "af_oth_male": 0.0,
      "af_popmax": 0.000123077,
      "af_raw": 1.1929e-05,
      "af_sas": 3.26755e-05,
      "af_sas_female": 0.0,
      "af_sas_male": 4.33614e-05
    }
  },
  "gnomad_genome": {
    "_license": "http://bit.ly/2I1cl1I",
    "af": {
      "af": 6.36821e-05,
      "af_afr": 0.00022941,
      "af_afr_female": 0.000271592,
      "af_afr_male": 0.00019857,
      "af_amr": 0.0,
      "af_amr_female": 0.0,
      "af_amr_male": 0.0,
      "af_asj": 0.0,
      "af_asj_female": 0.0,
      "af_asj_male": 0.0,
      "af_eas": 0.0,
      "af_eas_female": 0.0,
      "af_eas_male": 0.0,
      "af_female": 7.17978e-05,
      "af_fin": 0.0,
      "af_fin_female": 0.0,
      "af_fin_male": 0.0,
      "af_male": 5.72148e-05,
      "af_nfe": 0.0,
      "af_nfe_est": 0.0,
      "af_nfe_female": 0.0,
      "af_nfe_male": 0.0,
      "af_nfe_nwe": 0.0,
      "af_nfe_onf": 0.0,
      "af_nfe_seu": 0.0,
      "af_oth": 0.0,
      "af_oth_female": 0.0,
      "af_oth_male": 0.0,
      "af_popmax": 0.00022941,
      "af_raw": 6.36618e-05
    }
  }
}
```
